tests/Schema/Keywords/MaxPropertiesTest.php

```diff
@@ -28,7 +28,7 @@ public function testItValidatesMaxPropertiesRed() : void
     {
         $spec = <<<SPEC
 schema:
-  type: array
+  type: object
```

Comment:
Spec says it min\max properties should be validated against object only https://swagger.io/docs/specification/data-models/data-types/#property-count